meteor/server/api/playout/playout.ts

```diff
@@ -212,17 +221,27 @@ export namespace ServerPlayoutAPI {
 			let pBlueprint = makePromise(() => getBlueprintOfRundown(rundown))
 
 			const currentPart = rundown.currentPartId ? rundownData.partsMap[rundown.currentPartId] : undefined
-			if (currentPart && currentPart.transitionDuration) {
+			if (currentPart) {
 				const prevPart = rundown.previousPartId ? rundownData.partsMap[rundown.previousPartId] : undefined
 				const allowTransition = prevPart && !prevPart.disableOutTransition
+				const start = currentPart.getLastStartedPlayback()
 
 				// If there was a transition from the previous Part, then ensure that has finished before another take is permitted
-				if (allowTransition) {
-					const start = currentPart.getLastStartedPlayback()
+				if (allowTransition && currentPart.transitionDuration) {
 					if (start && now < start + currentPart.transitionDuration) {
 						return ClientAPI.responseError('Cannot take during a transition')
 					}
 				}
+
+				const offset = currentPart.getLastPlayOffset()
+				if (start && offset && currentPart.expectedDuration) {
+					const t = Date.now() - start! + offset!
+
+					// If there is an auto next planned
+					if (currentPart.autoNext && t > currentPart.expectedDuration - AUTOTAKE_DEBOUNCE) {
```

Comment:
I would change this to
`if (currentPart.autoNext && Math.abs(t - currentPart.expectedDuration) > AUTOTAKE_DEBOUNCE)`
to ensure that you _can_ do a Take even after an auto-take, in the (however unlikely) case the auto-take didn't fire